GliaWidgets/Lib/Screensharing/ScreenshareHandler.swift

```diff
@@ -0,0 +1,19 @@
+import SalemoveSDK
+
+class ScreenshareHandler {
+    private var visitorState: VisitorScreenSharingState?
+
+    func updateState(to state: VisitorScreenSharingState?) {
+        visitorState = state
```

Comment:
do we need to stop sharing before assign a new state?

Reply:
The SDK doesn't actually fire an event of screen sharing being ended (I would've assumed it would trigger an `onVisitorScreenSharingStateChange` with the status of `.notSharing`, but that's not the case). This is only used to signal the screen sharing session actually starting to show the "End screen share" button.

So yeah, I don't think it's needed.